DOC/source/datasheet/sofa_chd/custom_cells/custom_cells.rst

```diff
@@ -0,0 +1,133 @@
+=========================================================
```

Comment:
Can you register the rst file to the index file? As such, the rst will be included in the doctree. We can check the compiled version in CI.

Reply:
okay ill look into this